reference/pulseox-data-collector.v1.json

```
@@ -445,6 +454,9 @@
                 "timestamp"
               ]
             }
+          },
+          "height": {
```

I'd suggest to follow the same format as in the other fields you've added above (e.g., `Weight`, `Gyroscope`, etc.)
```suggestion
          Height": {
```